### DISCUSSION TITLE
Standalone DAG processor

### DISCUSSION THREAD

**Discussion by wpromatt** [apache/airflow#24424] [General] [answered]
### Official Helm Chart version

1.6.0 (latest released)

### Apache Airflow version

2.3.2 (latest released)

### Kubernetes Version

Any

### What happened

There is no service for running the new optional standalone dag processor.

### What you think should happen instead

There should be a configuration setting for enabling the standalone dag processor as documented here: https://airflow.apache.org/docs/apache-airflow/stable/concepts/dagfile-processing.html

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment, marked as answer]
You apparently have not noticed that #23711 is in progress. The best you can do is to help testing it by checking out that branch - will you help @wpromatt  ? Can we count on your help there?

**Comment by potiuk** [comment]
Generally - please do not open "bugs" for things that are not implemented yet. And if you want to speed up tnings - offering your help on things in progress is the best you can do.

**Reply by wpromatt** [reply]
My apologies. And will do!
